pages/reference.mdx

```diff
@@ -24,9 +24,10 @@ Discord exposes different versions of our API. You can specify which version to
 
 | Version | Status                           | Default |
 | ------- | -------------------------------- | ------- |
+| 10      | Available                        |         |
 | 9       | Available                        |         |
-| 8       | Available                        |         |
-| 7       | Doesn't look like anything to me |         |
+| 8       | Deprecated                       |         |
+| 7       | Deprecated                       |         |
 | 6       | Deprecated                       | ✓       |
```

Comment:
Shouldn't these be discontinued?

Reply:
They haven't discontinued them yet for reasons:tm: